src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/Models/EmptyElement.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Hl7.Fhir.ElementModel;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Serialization;
+using Hl7.Fhir.Specification;
+using Hl7.Fhir.Utility;
+using Microsoft.Health.Fhir.Anonymizer.Core.Models;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Fhir.Anonymizer.Shared.Core.Models
+{
+    public class EmptyElement: ITypedElement
+    {
+        public EmptyElement(string type)
+        {
+            InstanceType = type;
+        }
+        private static Meta _meta = new Meta() { Security = new List<Coding>() { SecurityLabels.REDACT } };
```

Comment:
nit: keep proper line spaces in this file (Line 18 & 25)

Reply:
Thanks, fixed it.